txpipe/sysmaps.py

```diff
@@ -152,7 +152,7 @@ def run(self):
             shear_tmp['dec'] = phot_data['dec']
             shear_psf_tmp['ra'] = phot_data['ra']       # Does it have 'ra' ?
             shear_psf_tmp['dec'] = phot_data['dec']     # Does it have 'dec' ?
```

Comment:
can you remove these changes so the PR as a whole is cleaner?
you can do this with git checkout to get the master version of this file.